src/components/PricingTableCard.svelte

```diff
@@ -0,0 +1,120 @@
+<script>
+  export let title = "";
+  export let description = "";
+  export let features = [];
+  export let buttonText = "";
```

Comment:
These props (button and pricing) seems too content-specific to be managed by the card component.

Is it possible to instead have the parent (pricing table) passes the footer elements (slots?) into the card component? This way, the parent could control the styling & layout of footer content directly.